VideoAPI/Video.API/Controllers/ConsultationController.cs

```diff
@@ -240,6 +246,55 @@ public async Task<IActionResult> StartPrivateConsultationWithEndpointAsync(Endpo
 
             return Accepted();
         }
+
+        [HttpPost("/consultations/start")]
+        [SwaggerOperation(OperationId = "StartPrivateConsultation")]
+        [ProducesResponseType((int) HttpStatusCode.Accepted)]
+        [ProducesResponseType((int) HttpStatusCode.NotFound)]
+        [ProducesResponseType((int) HttpStatusCode.Unauthorized)]
+        [ProducesResponseType((int) HttpStatusCode.BadRequest)]
+        [AllowAnonymous]
+        public async Task<IActionResult> StartConsultationRequestAsync(StartConsultationRequest request)
+        {
+            _logger.LogDebug("StartJudgeJohConsultationRequest");
```

Comment:
not needed added by middleware
same for 
`_logger.LogTrace("{ParticipantId} successfully transferred in consultation room: {roomLabel}", request.RequestedBy, room.Label);`